tools/molecule.spec

```
@@ -0,0 +1,59 @@
+%global srcname molecule
+%global pypiname molecule
+%global pkgname python3-%{srcname}
+%global forgeurl https://github.com/ansible-community/%{srcname}
+
+Name:    %{pkgname}
+Version: 0.0.0
+Release: 1%{?dist}
+Summary: ...
+%forgemeta
+URL:       %{forgeurl}
+Source:    %{forgesource}
+License:   BSD
```

https://github.com/ansible-community/molecule/blob/main/LICENSE
```suggestion
License:   MIT
```